packages/class-validator/src/__tests__/class-validator.test.ts

```diff
@@ -44,14 +59,26 @@ describe('class-validator', () => {
     id: 'c4a760a8-dbcf-4e14-9f39-645a8e933d74',
     name: 'John Doe',
     updatedAt: '2021-01-01T00:00:00.000Z',
-  };
+  } as Schema;
+
   const badData = {
     age: '123',
     createdAt: '2021-01-01T00:00:00.000Z',
     email: 'john.doe@test.com',
     id: 'c4a760a8-dbcf-4e14-9f39-645a8e933d74',
     name: 'John Doe',
+    updatedAt: '2021-01-01T00:00:00.000Z'
+  };
+
+  const badNestedData = {
+    age: 123,
+    createdAt: '2021-01-01T00:00:00.000Z',
+    email: 'john.doe@test.com',
+    id: 'c4a760a8-dbcf-4e14-9f39-645a8e933d74',
+    name: 'John Doe',
     updatedAt: '2021-01-01T00:00:00.000Z',
+    nested: new NestedSchema(),
+    nestedArray: [new NestedSchema()],
```

Comment:
required to create instances here, otherwise class-validator doesn't know how to validate.